### DISCUSSION TITLE
Switching characters through Order Select in Co-Op does not adjust for controls.

### DISCUSSION THREAD

**Discussion by KingPepe** [ikemen-engine/Ikemen-GO#937] [Feature Request] [answered]
**Describe the bug**
If order select is enabled for a co-op mode and you switch the characters around in Simul/Tag, the characters will be controlled by the wrong players.

**To Reproduce**
Detailed steps to reproduce the behavior, e.g.:
1. Enable order select for co-op modes under in main.lua. In my case, I changed "main.orderSelect" to be true for both values under "main.f_default()" which enables it for all modes regardless. Make sure the screenpack also has "orderselect.enabled" set to 1.
2. Select a co-op mode and then select any two characters.
3. On the versus screen, switch both characters around so that player 2's character becomes the first character and player 1's character becomes the second character.
4. When the match starts, player 1 will play as player 2's character. Player 2 will also be playing as player 1's character.

**Expected behavior**
Despite the order of the characters being switched, both players should be playing as the respective character they chose.

**Screenshots / Video**
[https://streamable.com/hjug16](https://streamable.com/hjug16)
First match shows the inputs. Second match shows the bug in action after the characters are switched.

**Platform (please complete the following information):**
 - 0.98.2 (latest release)
 - Windows 10

**Comment by K4thos** [comment, marked as answer]
This is not a bug since there is no such feature for co-op teams and it was never planned to be.
